src/SimpleCalculator.Console/UserConsoleInputValidation.cs

```diff
@@ -1,4 +1,5 @@
-namespace Calculations;
+using Calculations;
+namespace SimpleCalculator;
 
 public class UserConsoleInputValidation : IUserConsoleInputValidation
```

Comment:
It makes sense to rename `IUserConsoleInputValidation` interface

Reply:
Oh, absolutely. My bad!